PiicoDev_Unified.py

```diff
@@ -159,11 +153,24 @@ def read16(self, addr, reg):
         regInt = int.from_bytes(reg, 'big')
         return self.i2c.read_word_data(addr, regInt).to_bytes(2, byteorder='little', signed=False)
 
-def create_unified_i2c(bus=None, freq=None, sda=None, scl=None, suppress_warnings=True):
-    if _SYSNAME == 'microbit':
-        i2c = I2CUnifiedMicroBit(freq=freq)
-    elif _SYSNAME == 'Linux':
-        i2c = I2CUnifiedLinux(bus=bus, suppress_warnings=suppress_warnings)
+def create_unified_i2c(i2c_obj=None, bus=None, freq=None, sda=None, scl=None, suppress_warnings=True):
+    if i2c_obj is None:
```

Comment:
This would require modifications to each device driver to enable catching a `i2c_obj`.
Can we scope this for a separate feature?